src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -185,10 +168,24 @@ export const validationSchema = Yup.object()
     showReactions: Yup.bool().notRequired(),
     auditoriumColumns: Yup.number()
       .notRequired()
-      .min(5, "Columns must be at least 5"),
+      .min(
+        MINIMUM_COLUMNS,
+        `The columns need to be between ${MINIMUM_COLUMNS} and ${MAXIMUM_COLUMNS}.`
+      )
+      .max(
+        MAXIMUM_COLUMNS,
+        `The columns need to be between ${MINIMUM_COLUMNS} and ${MAXIMUM_COLUMNS}.`
+      ),
     auditoriumRows: Yup.number()
       .notRequired()
-      .min(5, "Rows must be at least 5"),
+      .min(
+        MINIMUM_COLUMNS,
+        `The rows need to be between ${MINIMUM_COLUMNS} and ${MAXIMUM_COLUMNS}.`
+      )
+      .max(
+        MAXIMUM_COLUMNS,
+        `The rows need to be between ${MINIMUM_COLUMNS} and ${MAXIMUM_COLUMNS}.`
+      ),
   })
```

Comment:
should be rows, not columns

Reply:
fixed in 5d2a552

Reply:
@malaikaaiyar they are about the partymap, correct? Would be good to rename those 4 to `MINIMUM_PARTYMAP_COLUMNS_COUNT`

Reply:
@mike-lvov ah I didn't see that. I will rename those variables and make new ones for the auditorium validation

Reply:
@malaikaaiyar we have the variable for the auditorium validation 
![image](https://user-images.githubusercontent.com/63194656/126673816-06729aba-56e0-47d4-8620-9d9e4427fe3c.png)

Reply:
fixed in 1416c33c